### ISSUE TITLE
Feedback request- changes to login modal window

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/arctos#9504]
I've gotten a lot of suggestions for updating the popup modal for logging in and it does need clarity and better flow. So here's a VERY SIMPLE proposed draft to address some of the most important ones (maybe not all!) So please add comments and we can go from there!

[Click on this link to leave comments in slideshow](https://docs.google.com/presentation/d/1ZlU6XY7hahDXJtH3wak1zu-CCiSjLQC2DxqnbU5k4I0/edit?usp=sharing) Or add comments below

<img width="960" height="540" alt="Image" src="https://github.com/user-attachments/assets/21d29166-7396-4f17-9ea8-5800187aed6d" />

(The blue box is just to have a div of a different color (not wedded to this shade at all) to distinguish between the general log in options and TACC. FYI, these boxes are dynamic so it is not stretched to edge of slide.)

**Comment by kderieg322079** [comment]
I think that looks good.

I have a comment more related to the function of accounts. It would be nice if there were a way for operators to search other collections for which they are not an operator. It is cumbersome to have to have an extra user account just to be able to search other collections. I know it is complex because of all of the tools you can use on the results table, but I'd love to see some way to be in "operator mode" and "user mode" while logged into the one account.

**Comment by Jegelewicz** [comment]
I am used to the page as it is - do we really have so many requests for change?

**Comment by atrox10** [comment]
The Welcome to Arctos font could be bigger, maybe have the highlighted color be the top (for non-Arctos users) and the non-highlighter are be for Arctos Users.  Or make the Arctos User color lighter. The issue I am having is outside users don't understand what to do when they try to see Arctos, so that should be highlighted whereas all of us in Arctos know to go to TACC.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This still seems really busy and confusing to me.

**Comment by ccicero** [comment]
I like @atrox10 suggestion. Eye should go first to "Welcome to Arctos" and the non-operator login options. Make the "Arctos Database Operators" block more subtle relative to non-operators.

**Comment by falco-rk** [comment]
>I'd love to see some way to be in "operator mode" and "user mode" while logged into the one account

Please. This would be amazing.

**Comment by campmlc** [comment]
Do we really need the words "but TACC users may use their account without database permissions" in the TACC login section? I'm confused by this wording, and not sure it is necessary here. Do we have a lot of TACC operators without database permissions trying to use this login? If it isn't necessary, eliminating those words would help simplify and clarify the screen. But I agree with the comments above to focus the login on being clear, simple, and obvious to non-users, at the top of the page, and then having a more discrete button for operator access at the bottom.

**Comment by dustymc** [comment]
>more discrete button for operator access at the bottom.

But that's not what it is. The TACC button isn't limited to operators (but operators are limited to TACC OAuth); anyone who can get a TACC account can use it to access Arctos.

**Comment by campmlc** [comment]
> But that's not what it is. The TACC button isn't limited to operators (but operators are limited to TACC OAuth); anyone who can get a TACC account can use it to access Arctos.

So TACC access can also be used for non-operator accounts? I didn't realize that. Confusing. So let's just leave the wording and hopefully anyone who needs that info can figure it out.
Again, I'm happy with the changes proposed [here](https://github.com/ArctosDB/arctos/issues/9504#issuecomment-4307262910)

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Is this bit necessary? Especially since logging is required now?

<img width="647" height="35" alt="Image" src="https://github.com/user-attachments/assets/038909c3-2e16-46f9-907d-97aa2b28eb7e" />

**Comment by ebraker** [comment]
Agree with comments above and would de-prioritize operator text and make it text small, and the welcome text large.

**Comment by atrox10** [comment]
I understand that TACC isn't limited to ARCTOS operators, but people already don't want to login to use a database. They are used to VertNet or other things where no log in required for searching, there's no way I can get users that want to borrow specimens/tissues to set up a TACC account with the MFA (unless they already had a TACC account, in which case none if this is a problem). General users are already having a hard enough time logging in with Google or a local account.

I agree that the text that says "To maximize your use of Arctos...." should just go away, since you HAVE to login now,
I would change it to something like this - 
**"To search on Arctos you need to login using one of these methods:"**
Then make the top stuff in light blue with a darker font, and de-emphasize the the TACC ARCTOS Data Operators Box.

We just need it more clear that you have to login to search Arctos.

**Comment by AJLinn** [comment]
> people already don't want to login to use a database. 

Curious where we are with getting a captcha-like option for non-logged in options that was mentioned a few months ago? I agree with @atrox10 that providing a non-login option is a high priority and is actively deterring collaboration.

**Comment by mkoo** [comment]
> Curious where we are with getting a captcha-like option for non-logged in options that was mentioned a few months ago?

I have not heard back from cloudflare and our application.
